### PR TITLE
New mobs to hassel everyone

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Fixes a bad pipe in atmos
Fixes bad pixel in deathclaws
Fixes bad paths in snake death
Adds a new nightmare type. Much deadly much rarer
Adds a new faction of monsters to beware of in the jungel, river forest and swamps
New faction is ape like birds that use simple rocks and spears at grate speeds to death with foes and when that fails use their brutal claws to nash and claw away the flesh.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/big.dm [hunk 1]
```diff
@@ -209,6 +209,22 @@
 	bones_amount = 30 //Lots of bone-like chitin
 	mob_size = MOB_LARGE
 
+/mob/living/simple_animal/hostile/nightmare/dream_daemon
+	name = "dream daemon"
+	desc = "Stalker of nightmare grown in darkness, a rarer mutation has made this verson much more deadly and stronger..."
```
Comment (ThePainkiller) [inline code/modules/mob/living/simple_animal/hostile/big.dm:214]:
```suggestion
	desc = "A creature of primal nightmares, bred in pitchest black... a rare sight of an evolved nightmare stalker, much deadlier, ruthless and agile than the average."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Some minor typo fixing

**Comment by Kazkin** [comment]
Where is the voxy.dmi sprites? I don't see them.

**Comment by Trilbyspaceclone** [comment]
was added before, they do infact work ill dm you them

**Pull request merged**